{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -9,6 +9,9 @@ argon2-cffi==19.2.0  # https://github.com/hynek/argon2_cffi
 whitenoise==5.0.1  # https://github.com/evansd/whitenoise
 {%- endif %}
 redis==3.5.0 # https://github.com/andymccurdy/redis-py
+{%- if cookiecutter.windows == "n" %}
```

Comment:
I think this condition should be also considering Docker, we can install `hiredis` if it's selected:

```suggestion
{%- if use_docker == "y" or cookiecutter.windows == "n" %}
```

We could also specify it as extra of redis:

```
redis[hiredis]
```

Reply:
I don't use a windows machine anymore, but I honestly wouldn't risk it. I wouldn't know if the build would succeed.